docs/en/enterprise-edition/content-collections/cloud-and-software-inventory/iac-resources.adoc

```diff
@@ -51,4 +58,36 @@ For the IaC Resources Inventory, filters provide a granular way to customize you
 * *Framework:* Allows you to focus on IaC assets based on the specific set of tools, libraries, and conventions used in their implementation. It provides insights into the underlying technology stack and methodology employed.
 * *IaC Resource Tag:* Offers insights into the tags associated with the IaC resource.
 * *Code Provider:* Indicates the cloud service provider associated with the asset.
-* *Last Modified By:* Use this filter to track and view changes made to the IaC asset based on the user who most recently modified it. It provides transparency in the development and maintenance process.
\ No newline at end of file
+* *Last Modified By:* Use this filter to track and view changes made to the IaC asset based on the user who most recently modified it. It provides transparency in the development and maintenance process.
+* *Resource Classes*: Filter IaC misconfigurations based on IaC resource classes. For more information, see <<resource-class,Resource Classes>> below.
+
+[#resource-class]
+=== Resource Classes
+
+IaC resource classes provide a structured approach to categorizing infrastructure resources based on their type, function, or other relevant criteria. This categorization enables efficient filtering and management of assets within the IaC inventory.
+
+NOTE: Currently, only Terraform and CloudFormation frameworks support IaC resource classes.
```

Comment:
```suggestion
NOTE: Currently, Terraform, CloudFormation, ARM and Bicep frameworks support IaC resource classes.
```

This will change by next week, so I think we shouldn't say just TF and CFN